### PR TITLE
AGP-70: Decentralize Nest

### PR DESCRIPTION
**Pull request by LouisGrx**

### CODE REVIEW

File: AGPs/AGP-X .md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+
+| AGP | Title | Author | Status | Track   | Created |
+|-----|-------|---------------------------|--------|---------|---------|
+| N/A  | Decentralize Nest |   louisgrx |      | Association | 2019-07-10       |
```
Comment (john-light) [inline AGPs/AGP-X .md]: Is the title "Decentralize Nest" or "Decentralizing Nest"?

File: AGPs/AGP-X .md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+
+
+# "AGP-X: Decentralizing Nest"
+
+## Description of desired Association policy change
+
+The Aragon Association is proposing the implemention of a Nest DAO governed by community members. This DAO will be in charge of selecting Nest grantees. We consider it a first step to decentralize Nest and make it an autonomous grant program.
```
Comment (john-light) [inline AGPs/AGP-X .md]:
```suggestion
The Aragon Association is proposing the implementation of a Nest DAO governed by community members. This DAO will be in charge of selecting Nest grantees. We consider it a first step to decentralize Nest and make it an autonomous grant program.
```

File: AGPs/AGP-X .md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Through this AGP, the Aragon Association is seeking approval from the community to take the necessary steps to fulfill this vision.
+
+## Motivation for changing this Association policy
+
+Up until now, the Aragon Association has been in charge of running the Nest program. This has included:
+1. Maintaining the Github Nest repository
```
Comment (john-light) [inline AGPs/AGP-X .md]:
```suggestion
1. Maintaining the Nest GitHub repository
```

File: AGPs/AGP-X .md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+2. Coordinating discussions around Nest proposals among network participants
+3. Selecting Nest teams and respectively allocating grants
+4. Following up on grantees, monitoring their deliverables, and ensuring coordination between grantees and the rest of the network
+
+The Aragon Association is proposing to let the Nest DAO in charge of  "3. Selecting Nest teams and respectively allocating grants" and to update the grant allocation process accordingly. The last specification for the new Nest grant allocation process can be found here: https://forum.aragon.org/t/agp-proposal-for-a-new-nest-grant-allocation-process/998.
```
Comment (john-light) [inline AGPs/AGP-X .md]:
```suggestion
The Aragon Association is proposing to put the Nest DAO in charge of  "3. Selecting Nest teams and respectively allocating grants" and to update the grant allocation process accordingly. The most recent specification for the new Nest grant allocation reflecting this new proposed process can be found here: https://forum.aragon.org/t/agp-proposal-for-a-new-nest-grant-allocation-process/998.
```

**Comment by LouisGrx** [comment]
This proposal is final and ready to be moved to later stages.

**Pull request merged**